templates/modules/b2b-account/quotes/quote-history.hypr.live

```diff
@@ -106,8 +123,7 @@
                                 {% set mobile=change.fields|findwhere("name", "Mobile") %}
 
                                 <td>
-                                  {% if change.type =='Add' %} {{labels.added}} {% elif change.type =='Remove' %} {{labels.removed}} {% else %} {{labels.updated}} {% endif %}
-                                  {{ changePath[0] }}
+                                  Address
```

Comment:
Add "Address"  into labels files (both en and de json). And FYI - if you add any new key in this file then add in alphabetical order. and then remove hardcoded label ("Address") and refer it from labels. see how we did in other places. ex . `{{labels.added}}`

Reply:
done